web/js/map-OpenLayers.js

```diff
@@ -115,6 +115,34 @@ var fixmystreet = fixmystreet || {};
         fixmystreet.markers.redraw();
       },
 
+      reorder_visible: function() {
+          var visible = ''
+          var $itemList = $('.item-list');
+          var $visibleReports = $('.visible-reports');
+          if ($visibleReports.length == 0) return false;
```

Comment:
Please always have brackets with if statements (I don't think jslint will let this through anyway :) ).
At the start of this function, you could do `if fixmystreet.page != 'reports' { return; }` as that variable is always set, to save any DOM lookups at all.